src/handlers/counting/repost/webhook.ts

```diff
@@ -23,9 +25,11 @@ export default async function repostWithWebhook(message: Message<true>, member:
     avatarURL: member.displayAvatarURL({ forceStatic: false, size: 64 }),
     allowedMentions: { parse: [], roles: [], users: []},
     ...channel.isThread() && { threadId: channel.id },
-  }).catch(() => {
+  }).catch((err: DiscordAPIError) => {
     // if it fails then assume the webhook is broken and delete it. if it's the second try however, just skip it.
     if (secondTry) return message;
+    const { guildId, channelId, id: messageId, author: { id: authorId }} = message;
+    countingLogger.error(`Failed to repost message in guild (${guildId}) [EMBED]:\n${inspect({ messageId, authorId, guildId, channelId, err })}`);
```

Comment:
See changes requested in the two other reposting files. This should be (webhook) as well fwiw